### ISSUE TITLE
API Version 2020-09-14

### ISSUE THREAD

**Issue by derekantrican** [Ackara/Plaid.NET#65]
According to the README, this library is configured for Plaid API version 2019-05-29. The latest Plaid API version is now 2020-09-14. Can you consider supporting this latest version?

**Comment by viceroypenguin** [comment]
@derekantrican I believe this library has been abandoned. I am maintaining Going.Plaid, which is current to the most recent documented updates to the API.

**Comment by derekantrican** [comment]
@viceroypenguin Thanks. Do you have a migration guide?

**Comment by viceroypenguin** [comment]
No, I don't have a migration guide unfortunately. You can find fresh installation information [here](https://github.com/viceroypenguin/Going.Plaid) or example code for using it [here](https://github.com/jcoliz/PlaidQuickstartBlazor/).
